calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,237 @@
+# marker_radar_lidar_calibrator
+
+In this tutorial, we present a hands-on tutorial of the `marker_radar_lidar_calibrator`. Although we provide a pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
+
+General documentation regarding this calibrator can be found [here](../../marker_radar_lidar_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../../README.md).
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1G1n2AV47gbQFKQtlzbHJPeV10m5_uMMk).
+
+The rosbag includes three different topics: `/sensing/radar/front_center/objects_raw`, `/sensing/lidar/front_lower/pointcloud_raw`, and `tf_static`.
+
+## Environment preparation
+
+### Overall calibration environment
+
+The required space for calibration depends on the vehicle and sensors used. Generally, an open space without any other objects is an ideal calibration environment. During the calibration, please always make sure that reflectors are detected by both radar and lidar. The user could always check whether the reflectors appear in the `rviz` with the corresponding topics.
```

Comment:
I have seen in other parts as well
It is not "the rviz", but "rviz"

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/f3ffbc4a72786c4bdb2df389a432428af40b0012

Reply:
I am pretty sure we discussed this just this week.
Comments that look that apply to more than one part should be checked through the whole repository.
A simple text based search indicates that this has not been done thoroughly

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/dd78b0a2bec348e59473562854c1ea0f79789b56